middleware/swagger-metadata.js

```diff
@@ -225,7 +225,7 @@ var processOperationParameters = function (swaggerMetadata, pathKeys, pathMatch,
 
   debug('  Processing Parameters');
 
-  async.map(_.reduce(parameters, function (requestParsers, parameter) {
+  var parsers = _.reduce(parameters, function (requestParsers, parameter) {
```

Comment:
Any reason we changed the existing approach?  Configuring multer shouldn't require that we change things around this much...but I could be wrong.